wp-content/themes/citylimits/inc/widgets/class-citylimits-special-projects-widget.php

```diff
@@ -0,0 +1,209 @@
+		$series_3_posts = citylimits_get_series_posts( $instance['series_3'], $instance['num'], 'featured, DESC' );
+
+		$series_arr = [$series_1_posts, $series_2_posts, $series_3_posts];
+
+		if ( empty( $series_1_posts ) && empty( $series_2_posts ) && empty( $series_3_posts ) ) return; //output nothing if no posts found
+
+		$widget_title = apply_filters( 'widget_title', empty( $instance['title'] ) ? '' : $instance['title'], $instance, $this->id_base );
+
+		echo $args['before_widget'];
+
+		echo '<div class="citylimits-special-projects-container">';
+
+		if ( ! empty( $widget_title ) ) echo $args['before_title'] . $widget_title . $args['after_title'];
+
+		$series_counter = 0;
+
+		echo '<div class="citylimits-special-projects-inner-container">';
+
+		foreach( $series_arr as $series ) {
+
+			$series_counter++;
+
+			if( $series->have_posts() ) {
+				
+				echo '<div class="citylimits-special-project">';
+
+				$thumbnail = '';
+				$series_id = $instance['series_'.$series_counter];
+				$title_link = get_term_link( (int) $instance['series_'.$series_counter], 'series' );
+				$term = get_term( $instance['series_'.$series_counter], 'series' );
+				$title = apply_filters( 'widget_title', $term->name, $instance, $this->id_base );
+				$excerpt = isset( $instance['excerpt_display'] ) ? $instance['excerpt_display'] : 'num_sentences';
+				
+				// get the term meta post so we can grab the featured image
+				$term_meta_post = largo_get_term_meta_post( 'series', $series_id );
+
+				if( largo_has_featured_media( $term_meta_post ) ){
+
+					$thumbnail = largo_get_featured_media( $term_meta_post );
+					$thumbnail = $thumbnail['attachment_data']['sizes']['large']['url'];
+
+					echo '<a class="citylimits-special-project-title" href="'.$title_link.'"><img class="citylimits-special-project-img" src="'.$thumbnail.'"><span class="citylimits-special-projects-title-border"><span>'.$title.'</span></span></a>';
+
+				} else {
+
+					echo '<a class="citylimits-special-project-title" href="'.$title_link.'"><div class="citylimits-special-project-img"></div><span class="citylimits-special-projects-title-border"><span>'.$title.'</span></span></a>';
+
+				}
+
+				echo '<h5 class="series-split top-tag">' . esc_html( $instance['heading'] ) .'</h5><ul>';
+
+				while ( $series->have_posts() ) {
+
+					$context = array(
+						'instance' => $instance,
+						'thumb' => false,
+						'excerpt' => $excerpt,
+					);
+
+					$series->the_post();
+					echo '<li>';
+					largo_render_template( 'partials/widget', 'content', $context );
+					echo '</li>';
+
+				}
+
+				echo '</ul></div>';
+
+			}
+
+		}
+
+		echo '</div></div>';
+
+		echo $args['after_widget'];
+
+		// Restore global $post
+		wp_reset_postdata();
+		$post = $preserve;
+	}
+
+	function update( $new_instance, $old_instance ) {
+		$instance = $old_instance;
+		$instance['title'] = strip_tags($new_instance['title']);
+		$instance['heading'] = sanitize_text_field( $new_instance['heading'] );
```

Comment:
What's the difference between Title and Heading?

Reply:
Heading wasn't being used, so removed in https://github.com/INN/umbrella-citylimits/pull/85/commits/15a450038f924d8c03060528b4945075367dffc9